test/unit/helpers.test.ts

```diff
@@ -39,3 +39,15 @@ describe("useEnv", () => {
     expect(process.env[envKey]).toEqual("test environment variable")
   })
 })
+
+describe("getAvailablePort", () => {
+  it("should return a valid port", async () => {
+    const port = await getAvailablePort()
+    expect(port).toBeGreaterThan(0)
+  })
+  it("should return different ports for different calls", async () => {
+    const portOne = await getAvailablePort()
+    const portTwo = await getAvailablePort()
+    expect(portOne).not.toEqual(portTwo)
```

Comment:
I'd add an `expect` that the ports are within the valid port range (0 to 65535)

Reply:
Good call! TIL there's a specific range!